packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/nodejs/sample-app.md

```diff
@@ -0,0 +1,109 @@
+
+With ngrok installed, a local application can now function as an external service, and demonstrate the Okta Event Hook (or any Okta Inline Hook). To get you up-and-running quickly, follow the steps below to build a very basic Express Node.js application. This application simply serves up a web page and responds to an Okta Event Hook.
+
+The Event Hook use-case is a simple response to the deactivation of an Okta user, which is also presented in [Event Hook Guide](/docs/guides/event-hook-implementation/overview).
+
+### Create a folder and initialize the project
+
+1. Create a local folder to hold your sample application and open it. In this example, `sample-app`.
+
+    ```bash
+    >sample-app
+    ```
+
+1. Initialize a default package to create a `package.json` file.
+
+    ```bash
+    >node init --yes
```

Comment:
For some reason, this command returned an error on my local machine (Mac Desktop) so I just skipped it and was able to bring up the sample HTML page. Here was the error I got:
![Screen Shot 2021-11-04 at 2 15 47 PM](https://user-images.githubusercontent.com/67654866/140421216-53c1409f-299c-43ce-babd-e08021044f64.png)

Reply:
Apologies, this should've been `npm init --yes` as the command; also neglected to mention to accept default values after running this command.